src/util/redis/redisConnector.test.js

```diff
@@ -2,8 +2,9 @@ const fs = require('fs');
 const path = require('path');
 const version = 'v0';
 const { RedisDB } = require('./redisConnector');
-const { structuredLogger: logger } = require('@rudderstack/integrations-lib');
+const logger = require('../../logger');
```

Comment:
No need of this import - check the reference, it's no longer required

Reply:
Done